doc/source/releases/21.01.rst

```diff
@@ -0,0 +1,878 @@
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 11416`_
+* Avoid false positives in Phylip sniffer.
+  `Pull Request 11595`_
+* Add display_in_upload for protobuf datatypes.
+  `Pull Request 11555`_
+* Don't display obs_names in select list.
+  `Pull Request 11574`_
+* Disable tool cache if there is a problem.
+  `Pull Request 10808`_
+* Put new toolshed tools in existing tool panel sections if the ID matches.
+  `Pull Request 10984`_
+* Make hash selection mandatory in hash tool.
+  `Pull Request 11427`_
+* Various Pyiron interactive tool fixes.
+  (thanks to `@bgruening <https://github.com/bgruening>`__).
+  `Pull Request 11545`_
+* Allow using Openrefine without data inputs.
+  (thanks to `@bgruening <https://github.com/bgruening>`__).
+  `Pull Request 11547`_
+* Set $HOME for interactive tools that require a specific directory.
+  (thanks to `@abretaud <https://github.com/abretaud>`__).
+  `Pull Request 11577`_
+* Add checks to workflow toolbox, avoid displaying empty sections.
+  `Pull Request 10363`_
+* Filter duplicate workflow output labels when initializing workflow step outputs.
+  `Pull Request 10432`_
+* Properly scale workflow node coordinates before saving workflow.
+  `Pull Request 10511`_
+* Fix changing versions in workflow editor.
+  `Pull Request 10516`_
+* Fix currently failing Selenium tests, move to GitHub workflow.
+  `Pull Request 10755`_
+* Don't preload editor mako with hidden workflows.
+  `Pull Request 10759`_
+* Fix generation of BCO `object_id` and `etag`
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 10964`_
+* Record invocation_step outputs as appropriate.
+  `Pull Request 10970`_
+* Fix workflow parameter text value restriction when connected to select and text parameter.
+  `Pull Request 10975`_
+* Convert deprecated `log.warn()`` calls to ``log.warning()``
+  (thanks to `@bgruening <https://github.com/bgruening>`__).
+  `Pull Request 10996`_
+* Misc fixes
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 11029`_
+* Maintain expanded state of JobStep during polling
+  `Pull Request 11144`_
+* Fix tool version switch in workflow editor.
+  `Pull Request 11257`_
+* Fix populated state for empty collections.
+  `Pull Request 11312`_
+* Fix Workflow report markdown lost upon renaming the workflow.
+  (thanks to `@davelopez <https://github.com/davelopez>`__).
+  `Pull Request 11317`_
+* Fix TypeError in WorkflowsAPIController.get_versions()
+  (thanks to `@rikeshi <https://github.com/rikeshi>`__).
+  `Pull Request 11373`_
+* Wrap core of workflow monitor thread in try/except.
+  `Pull Request 11434`_
+* Fix random-ish workflow error with maximum_workflow_jobs_per_scheduling_iteration.
+  `Pull Request 11513`_
+* Mark workflow as changed when editing step label or annotation.
+  `Pull Request 11533`_
+* Add input step data label when extracting workflow.
+  `Pull Request 11532`_
+* Fix saving FileSource parameter in a workflow.
```

Comment:
it's 20.09 fix. Do we also include it in 21.01 release notes?

Reply:
Yes, otherwise it is never listed in any release notes.

Reply:
> Yes, otherwise it is never listed in any release notes.

Ok. I see